src/renderer/components/reader/Reader.tsx

```diff
@@ -164,27 +170,26 @@ interface ReaderProps {
     reader?: any;
     mode?: any;
     infoOpen?: boolean;
-    deleteBookmark?: any;
-    addBookmark?: any;
-    findBookmarks: any;
+    deleteBookmark?: TReaderApiDeleteBookmark;
+    addBookmark?: TReaderApiAddBookmark;
+    findBookmarks: TReaderApiFindBookmarks;
     toggleFullscreen?: any;
     closeReader?: any;
     detachReader?: any;
-    setLastReadingLocation: any;
-    bookmarks?: LocatorView[];
+    setLastReadingLocation: TReaderApiSetLastReadingLocation;
+    bookmarks?: TReaderApiFindBookmarks_result;
     displayPublicationInfo?: any;
     publication?: Publication;
 }
-
 const defaultLocale = "fr";
 
 export class Reader extends React.Component<ReaderProps, ReaderState> {
     private fastLinkRef: any;
 
-    @lazyInject("store")
+    @lazyInject(diRendererSymbolTable.store)
     private store: Store<RootState>;
 
-    @lazyInject("translator")
+    @lazyInject(diRendererSymbolTable.history)
```

Comment:
Damn, I missed this fatal coding error (probably copy/paste mistake) during my PR review!
This line should be `@lazyInject(diRendererSymbolTable.translator)`
I am creating a quickfix into the `develop` branch.

Reply:
@panaC can the static TypeScript compiler checks verify that `diRendererSymbolTable.history` is not compatible with the type `Translator`? Right now the fatal crash is only detectable at runtime :(

Reply:
Hotfix: https://github.com/readium/readium-desktop/commit/12fcce9de927a37cc8b0c17731d78b3cc46546bb